py_expression_eval/tests.py

```diff
@@ -39,6 +39,12 @@ def test_parser(self):
         self.assertEqual(parser.parse('2*x + y').evaluate({'x': 4, 'y': 1}), 9)
         self.assertEqual(parser.parse("x||y").evaluate({'x': 'hello ', 'y': 'world'}), 'hello world')
         self.assertEqual(parser.parse("'x'||'y'").evaluate({}), 'xy')
+        self.assertEqual ( parser.parse ( "'x'=='x'" ).evaluate ( {} ), True )
```

Comment:
Awesome work! 
Can you please follow the coding style of the lines above.

Thanks!